chat-server/src/routes/conversations/addMessageToConversation.ts

```diff
@@ -101,108 +101,65 @@ export function makeAddMessageToConversationRoute({
       logRequest({
         reqId,
         message: stripIndents`Request info:
-        User message: ${message}
-        Stream: ${stream}
-        IP: ${ip}
-        ConversationId: ${conversationIdString}`,
+          User message: ${message}
+          Stream: ${stream}
+          IP: ${ip}
+          ConversationId: ${conversationIdString}`,
       });
 
-      let conversationId: ObjectId;
-      try {
-        conversationId = new ObjectId(conversationIdString);
-      } catch (err) {
-        return sendErrorResponse({
-          reqId,
-          res,
-          httpStatus: 400,
-          errorMessage: "Invalid conversation ID",
-        });
-      }
-
       if (!isValidIp(ip)) {
-        return sendErrorResponse({
-          reqId,
-          res,
+        throw makeRequestError({
           httpStatus: 400,
-          errorMessage: `Invalid IP address ${ip}`,
+          message: `Invalid IP address ${ip}`,
         });
       }
 
-      const shouldStream = Boolean(stream);
-      const latestMessageText = message;
+      const latestMessageText = message as string;
```

Comment:
Why the cast to string here?

Reply:
Holdover from ExpressRequest not being properly typed. Let me see if I can fix that...